systemtests/src/main/java/com/github/streamshub/systemtests/setup/StrimziOperatorSetup.java

```diff
@@ -0,0 +1,306 @@
+package com.github.streamshub.systemtests.setup;
+
+import com.github.streamshub.systemtests.Environment;
+import com.github.streamshub.systemtests.constants.Constants;
+import com.github.streamshub.systemtests.constants.Labels;
+import com.github.streamshub.systemtests.constants.ResourceKinds;
+import com.github.streamshub.systemtests.logs.LogWrapper;
+import com.github.streamshub.systemtests.utils.ResourceUtils;
+import com.github.streamshub.systemtests.utils.SetupUtils;
+import io.fabric8.kubernetes.api.model.ConfigMap;
+import io.fabric8.kubernetes.api.model.ConfigMapBuilder;
+import io.fabric8.kubernetes.api.model.ServiceAccount;
+import io.fabric8.kubernetes.api.model.ServiceAccountBuilder;
+import io.fabric8.kubernetes.api.model.apiextensions.v1.CustomResourceDefinition;
+import io.fabric8.kubernetes.api.model.apiextensions.v1.CustomResourceDefinitionBuilder;
+import io.fabric8.kubernetes.api.model.apps.Deployment;
+import io.fabric8.kubernetes.api.model.apps.DeploymentBuilder;
+import io.fabric8.kubernetes.api.model.rbac.ClusterRole;
+import io.fabric8.kubernetes.api.model.rbac.ClusterRoleBinding;
+import io.fabric8.kubernetes.api.model.rbac.ClusterRoleBindingBuilder;
+import io.fabric8.kubernetes.api.model.rbac.ClusterRoleBuilder;
+import io.fabric8.kubernetes.api.model.rbac.RoleBinding;
+import io.fabric8.kubernetes.api.model.rbac.RoleBindingBuilder;
+import io.fabric8.kubernetes.api.model.rbac.SubjectBuilder;
+import io.fabric8.openshift.api.model.operatorhub.v1.OperatorGroup;
+import io.fabric8.openshift.api.model.operatorhub.v1.OperatorGroupBuilder;
+import io.skodjob.testframe.resources.KubeResourceManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StrimziOperatorSetup {
+    private static final Logger LOGGER = LogWrapper.getLogger(StrimziOperatorSetup.class);
+    private static final KubeResourceManager RESOURCE_MANAGER = KubeResourceManager.get();
```

Comment:
out of curiosity, why do you have it as static here?

Reply:
For a more simple call usage for the whole class.

Reply:
I personally have it in the variable in `AbstractST` so in the STs I'm just doing `kubeResourceManager.create...`.
But in all other classes that are not in the `test` folder I'm using the `KubeResourceManager.get()`

I was mainly curious why you wanted to do it like that in the `StrimziOperatorSetup`. You want to do this in every class you will create or delete something?

(again, I'm just asking, if you want to do it like this, okay. I'm not sure if it can cause some issues or not.)

Reply:
Sure, let's keep it united everywhere, it might be better to call it then define it all around.